Microsoft.Toolkit.Uwp.Input.GazeControls/GazeFilePicker.xaml

```diff
@@ -0,0 +1,501 @@
+﻿<ContentDialog
+    x:Class="Microsoft.Toolkit.Uwp.Input.GazeControls.GazeFilePicker"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.Input.GazeControls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="Open File" 
```

Comment:
We'll want to localize these.

Reply:
Thanks. I will update. Is [this localization guidance](https://docs.microsoft.com/en-us/windows/uwp/app-resources/localize-strings-ui-manifest) still accurate? Or should I be following something else?

Reply:
Yup pretty much, we have some helpers in the Toolkit as well. You can see how we setup resources and used them for other controls in the Toolkit already.

Reply:
Fixed with the latest commit.